apps/docs/src/.vitepress/components/TopicOverviewCard.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+const props = defineProps<{
+  title: string;
+  subtitle: string;
+  image: string;
```

Comment:
just a suggestion, you decide :)
```suggestion
  imageSrc: string;
```